sdcm/cluster_k8s/__init__.py

```diff
@@ -2280,8 +2280,16 @@ def get_nodes_reboot_timeout(self, count) -> Union[float, int]:
     def get_nodes_readiness_delay(self) -> Union[float, int]:
         return self.PodContainerClass.pod_readiness_delay
 
-    def wait_for_pods_readiness(self, pods_to_wait: int, total_pods: int):
+    def wait_for_pods_readiness(self, pods_to_wait: int, total_pods: int, readiness_timeout: int = None):
```

Comment:
why calling it `readiness_timeout` instead of just `timeout`?

Reply:
to be aligned with the rest of the code that call it like that